src/Errors.ts

```diff
@@ -607,3 +613,17 @@ export function noTypesDefined() {
 export function tsConfigNotFound(cwd: string) {
   return `Grats: Could not find \`tsconfig.json\` searching in ${cwd}.\n\nSee https://www.typescriptlang.org/download/ for instructors on how to add TypeScript to your project. Then run \`npx tsc --init\` to create a \`tsconfig.json\` file.`;
 }
+
+export function noModuleInGqlExternal() {
+  return `Grats: @gqlExternal must include a module name in double quotes. For example: /** @gqlExternal "myModule" */`;
+}
+
+export function externalNotInResolverMapMode() {
+  return `Grats: @gqlExternal can only be used if grats is in EXPERIMENTAL__emitResolverMap mode. */`;
+}
+
+export function externalOnWrongNode(extistingTags: string[]) {
```

Comment:
I think we can just report this as an incompatibility with one tag and just report the first one that's incompatible. That will make the error message more readable. The plural case is probably not ever going to occur in practice, and if it does it's okay if it takes two rounds of errors for them to figure it out.